python/cuda_cccl/tests/parallel/test_transform.py

```diff
@@ -251,3 +252,68 @@ def op2(a):
     expected = np.arange(1, num_items + 1, dtype=np.int32) + 1
 
     np.testing.assert_allclose(expected, got)
+
+
+def test_unary_transform_well_known_negate():
+    """Test unary transform with well-known NEGATE operation."""
+    dtype = np.int32
+    d_input = cp.array([1, -2, 3, -4, 5], dtype=dtype)
+    d_output = cp.empty_like(d_input, dtype=dtype)
+
+    # Run unary transform with well-known NEGATE operation
+    parallel.unary_transform(d_input, d_output, parallel.OpKind.NEGATE, len(d_input))
+
+    # Check the result is correct
+    expected = np.array([-1, 2, -3, 4, -5])
+    np.testing.assert_equal(d_output.get(), expected)
+
+
+@pytest.mark.xfail(
+    reason="CCCL_IDENTITY well-known operation fails with NVRTC compilation error in C++ library"
+)
```

Comment:
Might be helpful to add an explanation as to why this happens and whether it will be fixed or not